generators/typescript-axios/exclusions.txt

```diff
@@ -1,2 +1,2 @@
-test/
+/test/
```

Comment:
This is needed as otherwise it matches the path of generated tests, i.e.:
`generated/artifacts/java/src/test/java/com/onfido/...`

Causing such tests not being removed when PR is generated ([example](https://github.com/onfido/onfido-node/pull/110/files#diff-d2e0bac7b8dd15c754fc431eccd67c2ea23189cf01769835d1b86ec01f837ac4)).